src/server/zset_family.cc

```diff
@@ -2599,6 +2646,301 @@ void ZSetFamily::GeoDist(CmdArgList args, ConnectionContext* cntx) {
                              distance_multiplier);
 }
 
+namespace {
+// Search all eight neighbors + self geohash box
+int MembersOfAllNeighbors(ConnectionContext* cntx, string_view key, const GeoHashRadius& n,
+                          const GeoShape& shape_ref, GeoArray* ga, unsigned long limit) {
+  vector<GeoHashBits> neighbors(9);
```

Comment:
Because you know the size in advance, you could use `std::array<GeoHashBits, 9>` instead of a `vector`. It would save a dynamic memory allocation